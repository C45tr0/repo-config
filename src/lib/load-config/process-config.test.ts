import processConfig from "./process-config";

test("processes top level config", () => {
  expect(
    processConfig({
      commands: {
        lint: ["yarn prettier", "yarn tslint2"]
      },
      filesToCopy: {
        "tsconfig.json": "files/tsconfig.json"
      }
    })
  ).toMatchSnapshot();
});

test("processes extension configs", () => {});
