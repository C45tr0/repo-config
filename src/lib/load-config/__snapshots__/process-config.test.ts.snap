// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`processes top level config 1`] = `
Object {
  "cmds": Object {
    "lint": Array [
      Object {
        "cmd": "yarn prettier",
        "loc": "local",
      },
      Object {
        "cmd": "yarn tslint2",
        "loc": "local",
      },
    ],
  },
  "files": Object {
    "tsconfig.json": "files/tsconfig.json",
  },
}
`;
